### PR TITLE
OOUX-AV

### PR DESCRIPTION
**Pull request by SondraE**
## Steps to test/reproduce
2021/11/11/object-oriented-ux/

### CODE REVIEW

File: content/blog/podcasts/object-oriented-ux.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  coming to CSS.
+media:
+  - iframe: https://open.spotify.com/embed/episode/7E4g4uJ7F6Bk6lpjj3grnZ
+---
+
+{% import "embed.macros.njk" as embed %}
+
+{{ embed.figure(media) }}
+
+<iframe width="100%" frameBorder="0" allowfullscreen="" allow="autoplay; clipboard-write; encrypted-media; fullscreen; picture-in-picture"></iframe>
```
Comment (mirisuzanne) [inline content/blog/podcasts/object-oriented-ux.md:37]:
```suggestion
```

Copy/paste leftovers?

**Review by mirisuzanne — COMMENTED** [review]
@SondraE looks like you just need a size on the iframe - sorry I missed that first time around

File: content/blog/podcasts/object-oriented-ux.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  ORCA stands for:
+  Objects, Relationships, CTAs, and Attributes.
+  We dive into the ways OOUX
+  facilitates a flow of conversation
+  between designers and back-end developers.
+  We get a sneak peak of Miriam's work on 
+  the new container queries features
+  coming to CSS.
+media:
+  - iframe: https://open.spotify.com/embed/episode/7E4g4uJ7F6Bk6lpjj3grnZ
```
[the remark this answers is not in the document]
Comment (SondraE) [reply content/blog/podcasts/object-oriented-ux.md:30]: Okay. Thanks!

File: content/blog/podcasts/object-oriented-ux.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
```
Comment (mirisuzanne) [inline content/blog/podcasts/object-oriented-ux.md:31]:
```suggestion
```

**Review by mirisuzanne — APPROVED** [review]

**Pull request merged**